jdk/src/share/classes/java/lang/ref/package.html

```diff
@@ -46,8 +46,7 @@ <h2>Package Specification</h2>
 references are for implementing memory-sensitive caches, weak references are
 for implementing canonicalizing mappings that do not prevent their keys (or
 values) from being reclaimed, and phantom references are for scheduling
-pre-mortem cleanup actions in a more flexible way than is possible with the
-Java finalization mechanism.
+scheduling post-mortem cleanup actions.
```

Comment:
Is the word "scheduling" repeated at both the end of unchanged line 48 and the beginning of new line 49?

Reply:
Good catch!   Both Kim and I missed this.